### DISCUSSION TITLE
Has the feature of hiding storys on the sidebar been implemented yet?

### DISCUSSION THREAD

**Discussion by TonyHoanTrinh** [storybookjs/storybook#15398] [Ideas]
Hello,

I wish to be able to hide stories on the sidebar and saw this thread (https://github.com/storybookjs/storybook/issues/9209) but I'm unsure if this feature has been implemented yet and if so how you can use it.

Thank you so much!

**Comment by shilman** [comment]
Open issue = issue has not been implemented
